src/js/tabs/adjustments.js

```diff
@@ -296,17 +304,15 @@ adjustments.adjust_template = function () {
     }
 
     // For 1.40, the D Setpoint has been replaced, so we replace it with the correct values
-
     const element22 = selectFunction.find("option[value='22']");
     const element23 = selectFunction.find("option[value='23']");
 
     // Change the "text"
     element22.text(i18n.getMessage('adjustmentsFunction22_2'));
     element23.text(i18n.getMessage('adjustmentsFunction23_2'));
```

Comment:
Actually we don't need to change `element22` or `element23`.  As the condition was recently removed please rename in `locales/en/message.json`:

`adjustmentsFunction22` should get the message defined in `22_2` (as `22_2` can then be removed)
Same with `adjustmentsFunction23`

Reply:
I remove the 2 lines.